src/Compilers/Core/Portable/Collections/DictionaryExtensions.cs

```diff
@@ -31,5 +31,24 @@ internal static class DictionaryExtensions
                 return value;
             }
         }
+
+#if !NETCOREAPP
```

Comment:
💡 Probably not necessary to make this conditional. The caller will bind different on .NET Core builds, but at runtime it would be fine to execute this if something did call it.

Reply:
This was an attempt to polyfill an API that wasn't available in .NET Framework. I'm not 100% sure it's needed anymore. The change has taken a couple of different shapes in the last few weeks.

Reply:
Yeah this isn't needed in the final change. Removing.